setup.py

```diff
@@ -1,7 +1,15 @@
+from pathlib import Path
 from setuptools import find_packages
 from setuptools import setup
 
-from detect_secrets.__version__ import VERSION
+
+def get_version():
+    """Parse package __version__.py to get version."""
+    versionpy = Path('detect_secrets/__version__.py').read_text()
```

Comment:
This should accommodate platforms that don't use `/` as a path separator:

```suggestion
    versionpy = Path('detect_secrets').joinpath('__version__.py').read_text()
```

Reply:
Fair point. Will ensure this gets added to the patch version.

Reply:
not sure if pathlib handles this by default anyway, but a more pythonic way would be to write `Path('detect_secrets') / '__version__.py'`. `/` is actually overloaded for path objects to do the right thing.